kustomize/components/clairpgupgrade/clair-pg-upgrade.job.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
 spec:
   template:
     spec:
-      restartPolicy: Never
+      restartPolicy: OnFailure
```

Comment:
Are you sure you won't need logs from previous attempts?

Reply:
This will only work for debugging, for production we need to restart the job if failed

Reply:
`restartPolicy: Never` means that the pod should be re-created by the job controller, not restarted by kubelet.
With `restartPolicy: Never` we potentially will have 50 (failed) pods with their logs. With `restartPolicy: OnFailure` we will only have the logs of the last attempt and the previous one. From QE results, the first attempt has important logs that we shouldn't loose.